@navikt/core/tokens/src/shadow.json

```diff
@@ -0,0 +1,36 @@
+{
+  "navds": {
+    "shadow": {
+      "xsmall": {
+        "value": "0px 2px 1px -1px rgb(0, 0, 0, 0.1), 0px 1px 3px 0px rgb(0, 0, 0, 0.1)"
+      },
+      "small": {
+        "value": "0px 2px 4px -2px rgb(0, 0, 0, 0.1), 0px 4px 6px -1px rgb(0, 0, 0, 0.1)"
+      },
+      "medium": {
+        "value": "0px 4px 6px -4px rgb(0, 0, 0, 0.1) ,0px 10px 15px -3px rgb(0, 0, 0, 0.1)"
+      },
+      "large": {
+        "value": "0px 8px 10px -6px rgb(0, 0, 0, 0.1), 0px 20px 25px -5px rgb(0, 0, 0, 0.1)"
+      },
+      "xlarge": {
+        "value": "0px 8px 10px -6px rgb(0, 0, 0, 0.1), 0px 25px 50px -12px rgb(0, 0, 0, 0.25)"
+      },
+      "inner": {
+        "value": "0px 2px 4px 0px rgb(0, 0, 0, 0.05) inset, 0px 1px 0px 0px rgb(0, 0, 0, 0.1) inset"
+      },
```

Comment:
Jeg kan ikke huske å ha sett denne i bruk noe sted. Er det noe nytt vi introduserer i det visuelle språket vårt?

Reply:
Den var en del av forslagene jeg satte opp. Om vi ikke har bruk for den, kan vi godt droppe den :)

Reply:
Om den har bruksområder, så støtter jeg å legge den til, men om ikke tenker jeg at det er unødvendig å gjøre ting mer komplisert enn de trenger.